twine/repository.py

```diff
@@ -132,7 +142,7 @@ def _upload(self, package):
                 allow_redirects=False,
                 headers={'Content-Type': monitor.content_type},
             )
-            bar.done()
+            bar.close()
```

Comment:
https://pypi.python.org/pypi/tqdm#manual leads me to believe that in fact `tqdm` can be used as a context manager.

Personally, I'd rather this look like it does except that we do:

```py
with bar:
    resp = self.session.post(
    # ...
    )
```

Because I'd rather keep the scope of the context-manager small.